### PR TITLE
(refactor) extract form loading steps from tests

### PR DESCRIPTION
**Pull request by jokeyrhyme**
- reduce at least 3 different ways we were loading form definitions down to 1 with assertions and 1 without
- updated tests 1-3, 5-24, 29-33
- all tests double-checked in Chrome and in PhantomJS

### CODE REVIEW

File: test/20_2013W31v/test.js [hunk 1]
```diff
@@ -1,52 +1,11 @@
 define(['BlinkForms', 'testUtils', 'BIC'], function (Forms, testUtils) {
```
Comment (simonmarklar) [inline test/20_2013W31v/test.js]: should this be completely removed ?

**Comment by jokeyrhyme** [comment]
We need the BIC module to be in the dep-tree somewhere so that `window.BMP.Forms.getDefinition()` is fixed up. However, I've made the testUtils module depend on that, and removed the BIC dependency from all the other modules.

File: test/15_renderSubForm/test.js [hunk 2]
```diff
@@ -130,39 +206,46 @@ define(['BlinkForms', 'testUtils', 'BIC'], function (Forms, testUtils) {
...
+                    Rank: '45',
+                    Status: ''
                   }
                 ]
               },
               {
                 id: '52',
                 Detail: 'Bad news',
-                Exp: [
+                Description: [
```
Comment (simonmarklar) [inline test/15_renderSubForm/test.js:228]: why did this change?

**Comment by simonmarklar** [comment]
oh i should of been a bit more clear (but i like that the test utils module loads bic now, one less thing to think about) - my question 

> should this be completely removed ?

was about the entire empty test suite https://github.com/blinkmobile/forms/blob/feature/tests-dedupe/test/20_2013W31v/test.js

**Comment by jokeyrhyme** [comment]
Oh, the purpose of that test suite is just to load a real-world form definition that used to not load at all.

File: test/15_renderSubForm/test.js (hunk 2 above)
Comment (jokeyrhyme) [inline test/15_renderSubForm/test.js:228]: It didn't. It's just git being dumb. Just the right number of lines were removed above to make it confused about 2 similar looking sections of code. The diff wouldn't be doing this if I'd done the un-nesting as a separate commit.

**Comment by simonmarklar** [comment]
oh ok that makes sense now. maybe add a comment that says that ? otherwise

lgtm :+1:

**Comment by jokeyrhyme** [comment]
I added that comment to test/20_2013W31v

**Pull request merged**